packages/react-devtools/package.json

```diff
@@ -1,7 +1,9 @@
 {
   "name": "react-devtools",
+  "productName": "React Developer Tools",
   "version": "2.0.12",
   "description": "Use react-devtools outside of the browser",
+  "main": "./packages/react-devtools/app.js",
```

Comment:
I think this should stay pointing at `index.js` (which it does by default when the key is absent). Is there a reason why this doesn't work? When you run from the command line [the bin script runs instead](https://github.com/facebook/react-devtools/blob/master/packages/react-devtools/bin.js) anyway.